docs/content/concepts/visualizers-and-overrides.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Visualizer and Overrides
+order: 650
+---
+
+This section explains the process by which logged data is used to produce a visualization and how it can be customized via the user interface or code.
+
+*Note*: this area is under heavy development and subject to changes in future releases.
+
+## How are visualizations produced?
+
+<!-- schematics source: https://excalidraw.com/#json=8G274_acK-zYc7Cq2ONf0,GaIabh3FBulcjNx9ZqJrXg -->
+
+<picture>
+  <img src="https://static.rerun.io/viscomp-base/dbe3202b78ff44e35dab0844b3494e3a68c7e551/full.png" alt="">
+  <source media="(max-width: 480px)" srcset="https://static.rerun.io/viscomp-base/dbe3202b78ff44e35dab0844b3494e3a68c7e551/480w.png">
+  <source media="(max-width: 768px)" srcset="https://static.rerun.io/viscomp-base/dbe3202b78ff44e35dab0844b3494e3a68c7e551/768w.png">
+  <source media="(max-width: 1024px)" srcset="https://static.rerun.io/viscomp-base/dbe3202b78ff44e35dab0844b3494e3a68c7e551/1024w.png">
+  <source media="(max-width: 1200px)" srcset="https://static.rerun.io/viscomp-base/dbe3202b78ff44e35dab0844b3494e3a68c7e551/1200w.png">
+</picture>
+
+In the Rerun viewer, visualizations happen within _views_, which are defined by their [_blueprint_](blueprint.md).
+
+The first step for a view to display its content is to determine which entities are involved.
+This is determined by the [entity query](../reference/entity-queries.md), which is part of the view blueprint.
+The query is run against the data store to generate the list of view entities.
+
+Views rely on visualizers to display each of their entities. For example, [3D views](../reference/types/views/spatial3d_view.md) use the `Points3D` visualizer to display 3D point clouds, and [time series views](../reference/types/views/time_series_view.md) use the `SeriesLine` visualizer to display time series line plots. Which visualizers are available is highly dependent on the specific kind of view. For example, the `SeriesLine` visualizer only exist for time series views—not, e.g., 3D views.
```

Comment:
```suggestion
Views rely on visualizers to display each of their entities. For example, [3D views](../reference/types/views/spatial3d_view.md) use the `Points3D` visualizer to display 3D point clouds, and [time series views](../reference/types/views/time_series_view.md) use the `SeriesLine` visualizer to display time series line plots. Which visualizers are available is highly dependent on the specific kind of view. For example, the `SeriesLine` visualizer only exist for time series views — not, e.g., for 3D views.
```

Reply:
This is actually a em-dash (not visible with monospace font) and I exploit the lack of style guide to decide that we dont use spaces around it :) A cursory look indicates that I'm mostly the only one to actually use them, so I'm not infringing on some untold consistency rule 😅 

Here is how it looks IRL (I'm taking that "for" in):
<img width="496" alt="image" src="https://github.com/rerun-io/rerun/assets/49431240/a4ac90a0-727f-413b-9f59-70b34ab56b76">

Reply:
Background: https://www.merriam-webster.com/grammar/em-dash-en-dash-how-to-use (last paragraph covers spacing)

Reply:
🤯